### DISCUSSION TITLE
StreetComplete iOS App

### DISCUSSION THREAD

**Discussion by ghost** [streetcomplete/StreetComplete#5230] [Ideas]
Street Complete iOS App discussion post for issue #1892

**Comment by BarterClub** [comment]
Yes please

**Reply by ghost** [reply]
I heard funding is an issue as well as the code transition. But funding is easily solved through community donations as 100$ for apples requirement can be raised

**Comment by ghost** [comment]
Would like to contribute bit I am primarily a iOS user

**Reply by ghost** [reply]
Would you be able to financial contribute? If they setup a fund?

**Reply by westnordost** [reply]
That's great! If you are a programmer, here is some information on how you could contribute: https://github.com/streetcomplete/StreetComplete/issues/1892#issuecomment-1671074967
